### ISSUE TITLE
Please update security document for 2.7

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#415]
Originally reported of Google Code with ID 359.

```
Confd now answers both network queries and master queries.
security.rst should be updated with more information on it.

Thanks,

Guido

```
Originally added on 2013-02-06 10:31:11 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Not only confd is undocumented, but also the new restricted commands. I suspect more work is needed to get this document really up-to-date for 2.7
```
Originally added on 2013-02-06 15:19:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-02-06 15:20:04 +0000 UTC.

Added Labels: Type-Documentation
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Patch pushed, I hope it's good enough.
```
Originally added on 2013-02-11 10:29:32 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
